Directory.Build.targets

```diff
@@ -21,4 +21,15 @@
     -->
     <NETCoreAppMaximumVersion>$(MajorVersion).$(MinorVersion)</NETCoreAppMaximumVersion>
   </PropertyGroup>
+
+  <!-- Packaging -->
+  <PropertyGroup>
+    <PackageDescription Condition="'$(PackageDescription)' == '' and '$(Description)' != ''">$(Description)</PackageDescription>
+    <PackageDescription Condition="'@(PackageDescriptionCommonType)' != '' and '$(PackageDescription)' != ''">$(PackageDescription)%0A%0ACommonly Used Types:@(PackageDescriptionCommonType->'%0A%(Identity)')</PackageDescription>
```

Comment:
PackageDescriptionCommonType itemgroup be empty during static evaluation ? as properties as evaluated before items ?

Reply:
right, properties are evaluated before items.